src/azure-cli/azure/cli/command_modules/mysql/_help.py

```diff
@@ -754,3 +754,11 @@
   - name: Resets GTID '3E11FA47-71CA-11E1-9E33-C80AA9429562:23' on server 'testsvr'.
     text: az mysql flexible-server gtid reset -g testgroup -s testsvr --gtid-set 3E11FA47-71CA-11E1-9E33-C80AA9429562:23
 """
+
+helps['mysql flexible-server export create'] = """
```

Comment:
Why don't you call it `mysql flexible-server export-backup create`?

I noticed that the client name and your description are all about export backup, not export